libs/coin-evm/docs/evm-family-integration-process/README.md

```diff
@@ -0,0 +1,106 @@
+  - Sending and receiving coins
+  - Viewing transaction history
+  - Token support, enabling interactions with the blockchain tokens
+- Use of public RPC node and explorer (etherscan or blockscout-like). This should be specified in the corresponding Jira Epic
+  - To find a public node, you can use https://chainlist.org/ for example
+  - To find a public explorer, you can use https://blockscan.com/ (etherscan-like) or https://www.blockscout.com/
+- The [Ethereum embedded app](https://github.com/LedgerHQ/app-ethereum) should handle this new network
+
+## Live codebase update
+
+Here is an example PR of an EVM currency integration: https://github.com/LedgerHQ/ledger-live/pull/4008
+
+### Main steps
+
+_Common steps for all new EVM currency integration_
+
+1. Add a new config entry for the new currency under [`libs/ledgerjs/packages/cryptoassets/src/currencies.ts`](https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledgerjs/packages/cryptoassets/src/currencies.ts)
+2. Add the new currency ID to the `CryptoCurrencyId` type under [`libs/ledgerjs/packages/types-cryptoassets/src/index.ts`](https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledgerjs/packages/types-cryptoassets/src/index.ts)
+3. Add an entry for the new currency in the `abandonSeedAddresses` (using the currency ID as key and `EVM_DEAD_ADDRESS` as value) under [`libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts`](https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts)
+4. Add the new currency ID to the `setSupportedCurrencies` function param on each relevant project ([CLI](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/cli/src/live-common-setup-base.ts), [LLD](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-desktop/src/live-common-set-supported-currencies.ts), [LLM](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-mobile/src/live-common-setup.ts) and [LLC test environement](https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/__tests__/test-helpers/environment.ts))
+5. Add a new feature flag config for this currency:
+   1. The new feature flag type in the [`CurrencyFeatures`](https://github.com/LedgerHQ/ledger-live/blob/7513354754ce0326a4ebcbcd86a5b4b38898a49e/libs/ledgerjs/packages/types-live/src/feature.ts#L80-L124) type under [`libs/ledgerjs/packages/types-live/src/feature.ts`](https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledgerjs/packages/types-live/src/feature.ts)
+   2. The new feature flag definition with default value in the [`CURRENCY_DEFAULT_FEATURES`](https://github.com/LedgerHQ/ledger-live/blob/7513354754ce0326a4ebcbcd86a5b4b38898a49e/libs/ledger-live-common/src/featureFlags/defaultFeatures.ts#L23-L67) mapping under [`libs/ledger-live-common/src/featureFlags/defaultFeatures.ts`](https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/featureFlags/defaultFeatures.ts)
```

Comment:
Will just need to update the file path when https://github.com/LedgerHQ/ledger-live/pull/5341 is merged

Reply:
Indeed 👍 
This doc (as it's the case for any doc) should be regularly updated to closely fit the current state of the codebase